receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml

```diff
@@ -58,6 +58,15 @@ metrics:
       monotonic: true
     attributes: [state]
 
+  process.cpu.utilization:
+    enabled: false
+    description: >-
+      Percentage of total CPU time used by the process since last scrape.
+      On the first scrape, no data point is emitted for this metric.
+    unit: 1
+    gauge:
+      value_type: double
```

Comment:
Why not per state?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md#metric-instruments

Reply:
The total is what [Telegraf](https://github.com/influxdata/telegraf/blob/v1.24.1/plugins/inputs/procstat/procstat.go#L234) reports and what the [gopsutil](https://github.com/shirou/gopsutil/blob/v3.22.8/process/process.go#L234) package exposes.

If we want process CPU utilization by state, we would need to either propose this addition to the `gopsutil` package or implement it in the `hostmetrics` receiver.

Reply:
Sure, but gopsutil calculates this from times https://github.com/shirou/gopsutil/blob/ed37dc27a286a25cbe76adf405176c69191a1f37/cpu/cpu_aix_cgo.go#L12, which means you can do the same calculation at state level, is just that you need to re-implement a bit the percent func.

Reply:
Sure, I could implement this. I wonder what @TylerHelmuth as the assignee or @dmitryax as the code owner think? Can you folks chime in?

Reply:
As per the spec says, we should do that.

Reply:
I finally started working on it, should have it done soon.

Reply:
This is now implemented, @bogdandrutu please take a look.